setup.py

```diff
@@ -6,7 +6,7 @@ def read(fname):
 
 setup(
 	name="fastly-python",
-	version="1.1.0",
+	version="1.2.0",
 	author="Chris Zacharias",
```

Comment:
is there a way to change the version that would be compatible with contributing this back to the original repo? it doesn't seem ideal that we do this in our fork.

Reply:
I don't think there's a simple way of accomplishing that @mloudon.
I think we would depend on the remote's release process.

Initially I think this was forked because the original looks discontinued.
What do you suggest?

Reply:
if the original is discontinued that seems fine.